components/FeatureProject/Thumbnail.tsx

```diff
@@ -0,0 +1,36 @@
+'use client';
+
+import Image from 'next/image';
+import type { Image as SanityImage } from 'sanity';
+
+import { urlForImage } from '@/sanity/lib/image';
+import { useRef } from 'react';
+import useIntersectionObserver from '@/hooks/useInterSectionObserver';
+
+type ThumbnailProps = {
+	thumbnail: SanityImage;
+	title: string;
+};
+
+export default function Thumbnail({ title, thumbnail }: ThumbnailProps) {
+	const ref = useRef<HTMLImageElement>(null);
+
+	useIntersectionObserver(
+		ref,
+		title === 'Morrent' ? ['animate-fade-left'] : ['animate-fade-right']
+	);
+
+	return (
+		<Image
+			ref={ref}
+			src={urlForImage(thumbnail).url()}
```

Comment:
Why do we have to do this?

Is it not possible to get the image URL directly when we fetch data from sanity?